src/styles/blocks/variables.scss

```diff
@@ -0,0 +1,6 @@
+$main-color: #060B35;
+$secondary-color: #616070;
+$blue-accent: #00ACDC;
+$white-accent: #fff;
+$card-border-color: #F3F3F3;
```

Comment:
please write all colors in lowercase like #f3f3f3, don't forget about this in your future tasks